ext/standard/basic_functions.stub.php

```diff
@@ -1152,7 +1152,7 @@ function password_algos(): array {}
  * @param array $pipes
  * @return resource|false
  */
-function proc_open(array|string $cmd, array $descriptorspec, &$pipes, ?string $cwd = null, ?array $env = null, ?array $other_options = null) {}
+function proc_open(array|string $command, array $descriptor_spec, &$pipes, ?string $cwd = null, ?array $env = null, ?array $options = null) {}
```

Comment:
maybe `$env_vars` (`$env` seems to be ok too)

Reply:
Done